lambdas/build-cri-oauth-request/src/test/java/uk/gov/di/ipv/core/buildcrioauthrequest/helpers/AuthorizationRequestHelperTest.java

```diff
@@ -168,6 +170,7 @@ void shouldThrowExceptionWhenUnableToSignJwt() {
                                         OAUTH_STATE,
                                         TEST_USER_ID,
                                         TEST_JOURNEY_ID,
+                                        null,
```

Comment:
Might be worth adding something into one of these tests, or creating a new one, that asserts that context gets added.